src/Browscap/Coverage/Processor.php

```diff
@@ -88,7 +77,7 @@
      * A temporary storage for coverage information for a specific file that is later merged into the main $coverage
      * property after the file is done processing.
      *
-     * @var array
+     * @var (string|array)[]
```

Comment:
This type doesn't make sense, it should either be a `string`, a `string[]` or an `array`...

Reply:
It is an array which elements may be string or array.

Reply:
Just use `array` then please ;)

Reply:
changed